packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -280,11 +293,14 @@ class EnrollVerbHandler extends AbstractVerbHandler {
     if (_doesEnrollmentHaveManageNamespace(enrollDataStoreValue)) {
       await _fetchAllEnrollments(enrollmentKeysList, enrollmentRequestsMap);
     } else {
-      enrollmentRequestsMap[enrollmentKey] = {
-        'appName': enrollDataStoreValue.appName,
-        'deviceName': enrollDataStoreValue.deviceName,
-        'namespace': enrollDataStoreValue.namespaces
-      };
+      if (!(enrollDataStoreValue.approval!.state ==
```

Comment:
use != instead of !(...==...)

Reply:
changed in [04c08a2](https://github.com/atsign-foundation/at_server/pull/1536/commits/04c08a2a6c3c5185aaccc7f02f1cdac7ea55d1eb)